portal/src/main/webapp/WEB-INF/jsp/global/header_bar.jsp

```diff
@@ -65,6 +65,37 @@
 }
 </style>
 
+<script type="text/javascript">
+function openSoicalAuthWindow() {
+    var _window = open('login.jsp', '', 'width=800, height=600');
+
+    var interval = setInterval(function() {
+        try {
+            if (_window.closed) {
+                clearInterval(interval);
+            } else if (_window.document.URL.includes(location.origin) &&
+                                !_window.document.URL.includes(location.origin + '/auth') &&
+                                !_window.document.URL.includes('login.jsp')) {
+                _window.close();
+
+                setTimeout(function() {
+                    clearInterval(interval);
+                    if(window.location.pathname.includes('/study')) {
+                        $('#rightHeaderContent').load(' #rightHeaderContent');
+                        iViz.vue.manage.getInstance().showSaveButton= true
+                    } else {
+                        location.reload();
+                    }
+                }, 500);
+            }
+        } catch (err) {
+            console.log('Error while monitoring the Login window: ', err);
```

Comment:
We should clear the interval here

Reply:
clearing the interval will not close the popup authentication window once the user login.